pipelines/DRP.yaml

```diff
@@ -1,16 +1,158 @@
+                                                   'z': 'phot_g_mean',
+                                                   'Y': 'phot_g_mean',
+                                                   'N419': 'phot_g_mean',
+                                                   'N540': 'phot_g_mean',
+                                                   'N708': 'phot_g_mean',
+                                                   'N964': 'phot_g_mean',
+                                                  }
+        config.astrometryRefObjLoader.anyFilterMapsToThis = None
+  makeWarp:
+    class: lsst.pipe.tasks.makeCoaddTempExp.MakeWarpTask
+    config:
+    - connections.photoCalibName: jointcal
+      doWriteEmptyWarps: true
+      python: 'from lsst.pipe.tasks.selectImages import PsfWcsSelectImagesTask;
+        config.select.retarget(PsfWcsSelectImagesTask)
+        
+        '
+subsets:
+  processCcd:
+    subset:
+    - isr
+    - characterizeImage
+    - calibrate
+    description: 'An alias of ProcessCcd to use in higher-level pipelines.
+      
+      '
+  step0:
+    subset:
+    - isrForCrosstalkSources
+    description: 'Tasks which should be run once, prior to initial data processing.
+
+      This step generates crosstalk sources for ISR/inter-chip crosstalk by applying 
+      overscan correction on raw frames. A new dataset is written, which should be 
+      used as an input for further data processing.
+      
+      '
+  step1:
+    subset:
+    - isr
+    - characterizeImage
+    - calibrate
+    - transformSourceTable
+    - writeSourceTable
+    description: 'Per-detector tasks that can be run together to start the DRP pipeline.
+
+      These should never be run with ''tract'' or ''patch'' as part of the data ID
+      expression if any later steps will also be run, because downstream steps require
+      full visits and ''tract'' and ''patch'' constraints will always select partial
+      visits that overlap that region.
+      
+      '
+  step2:
+    subset:
+    - consolidateSourceTable
+    - consolidateVisitSummary
+    description: 'Per-visit tasks that can be run together, but only after ''step1''.
+
+      These should never be run with ''tract'' or ''patch'' as part of the data ID
+      expression.
+      
+      '
+  step3:
+    subset:
+    - selectGoodSeeingVisits
+    - jointcal
+    - makeWarp
+    - templateGen
+    - deblend
+    - detection
+    - writeObjectTable
+    - measure
+    - mergeMeasurements
+    - assembleCoadd
+    - forcedPhotCoadd
+    - transformObjectTable
+    - consolidateObjectTable
+    - mergeDetections
+    description: 'Tract-level tasks that can be run together, but only after the ''step1''
+      and ''step2'' subsets.
+
+      These should be run with explicit ''tract'' constraints essentially all the
+      time, because otherwise quanta will be created for jobs with only partial visit
+      coverage.
+      
+      '
+  step7:
```

Comment:
I'll just point out that you went from step 3 to step 7... 🙃

Reply:
Yes - I mentioned on the ticket that this pipeline does not include steps 4 through 6 as in obs_subaru, as those subsets are not currently required for DECam DRP processing. Nevertheless, I have chosen to maintain the step N naming conventions for consistency with HSC data processing. For this reason, we jump from step 3 to step 7. I wanted to maintain the flexibility of potentially adding-in step 4/5/6 at some point in the future (if needs be) without needing to renumber the final wrap-up step.

Reply:
Got it, thanks. You can probably guess what I'll say next. A comment to that effect in the pipeline would be great! For the inevitable future when somebody asks on community, "how do I do DRP step 5 for DECam?"

Reply:
A brief in-line comment has been added to this effect.